.gitignore

```diff
@@ -23,3 +23,4 @@ coverage
 
 tests/certs/*.pem
 tests/certs/certs.state
+.npmrc
```

Comment:
Is it just this change?

Reply:
yeah I was doing something else, but this change was much simpler